README.md

```diff
@@ -12,6 +12,13 @@ Supports both TIFF and BigTIFF files. The underlying IO mechanisms are expected
   let tiff = Tiff.from_file (File.pread_exact r) in
   let ifd = Tiff.ifd tiff in
   let entries = Tiff.Ifd.entries ifd in
+  let data = Tiff.data tiff (File.pread_exact r) (Tiff.Data.UINT8) in 
+  let sum =
+    match data with
+    | UInt8Data data -> Owl.Dense.Ndarray.Generic.sum' data
+    | _ -> raise Tiff.Data.TiffDataHasWrongType
```

Comment:
Instead of a custom exception, let's just raise `Invalid_argument "Tiff Data has Wrong Type"`